src/SDKs/Graph.RBAC/Graph.RBAC.Tests/Graph.RBAC.Tests.csproj

```diff
@@ -28,4 +28,7 @@
   <ItemGroup>
     <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.7.4" />
+  </ItemGroup>
```

Comment:
included this why?

Reply:
I had to install this locally to use some of the functions in the class. In general, should this not be included?

Reply:
cant we get this from the common dependencies,so that our code wont be left behind when the sdk team updates

Reply:
I'll look into it.